exercises/palindrome-products/.meta/template.j2

```diff
@@ -0,0 +1,49 @@
+"""	
+Notes regarding the implementation of smallest and largest:
+
+Both functions must take two keyword arguments:
+    max_factor -- int
+    min_factor -- int, default 0
+
+Their return value must be a tuple (value, factors) where value is the
+palindrome itself, and factors is an iterable containing both factors of the
+palindrome in arbitrary order.
+"""
```

Comment:
This comment should be moved to `.meta/hints.md`, and the README will need to be [regenerated](https://github.com/exercism/python/blob/master/CONTRIBUTING.md#generating-exercise-readmes).